experimental/PropertyDDS/packages/property-common/package.json

```diff
@@ -54,10 +54,11 @@
 		"temp-directory": "nyc/.nyc_output"
 	},
 	"dependencies": {
+		"@types/events_pkg": "npm:@types/events@^3.0.0",
```

Comment:
Move to DevDeps, right?

Reply:
There is some debate about the right approach for this.
Dependents don't install devDeps and thus won't have types without it being installed. So making a prod dep makes it available and gets the right version of typing. This is probably the right thing to do for any types that are exported.
The downside is that in a dynamic environment this additional package will be installed. Don't think we care for this package and the type package is tiny.
This isn't technically required for this package as all of the dependencies on EventEmitter from here are .JavaScript and not TypeScript. :(

Reply:
> This is probably the right thing to do for any types that are exported.

Fair point.

> This isn't technically required for this package as all of the dependencies on EventEmitter from here are .JavaScript and not TypeScript. :(

Fair counterpoint 😄 .